contracts/pool/PoolFactory.sol

```diff
@@ -60,7 +60,7 @@ contract PoolFactory is IPoolFactory, Pausable, ReentrancyGuard {
         IHeliosGlobals _globals = globals;
         require(_globals.isValidPoolDelegate(msg.sender), "PF:NOT_DELEGATE");
 
-        _isMappingKeyValid(poolId);
+        // _isMappingKeyValid(poolId);
```

Comment:
Previously we added this line to avoid `poolId` duplication (from backend perspective).
Could you share your reasoning behind commenting this line with us?